rllib/core/examples/simple_ppo_rl_module.py

```diff
@@ -309,7 +309,7 @@ def _forward_train(self, batch: NestedDict) -> Mapping[str, Any]:
             mu, scale = pi_out.chunk(2, dim=-1)
             action_dist = TorchDiagGaussian(mu, scale.exp())
 
-        logp = action_dist.logp(batch["action"].squeeze(-1))
+        logp = action_dist.logp(batch["actions"].squeeze(-1))
         entropy = action_dist.entropy()
```

Comment:
needed to do this to make compatible with sample batch

Reply:
Great catch. Noticed that, too, in another PR (the gym one).
Even better, can we use the `SampleBatch.ACTIONS` constant here and for the other keys in this file?